README.md

```diff
@@ -272,7 +272,7 @@ charger_plus_car_roundtrip_efficiency: 85
 ## ALWAYS CHECK/CHANGE ##
 
 # The maximum energy storage capacity of the battery of the car, as an integer.
-# For the Nissan Leaf this is usually 24, 40 or 62
+# For the Nissan Leaf this is usually 21, 36 or 56. See https://ev-database.org.
```

Comment:
This might list each value next to the brochure value (which I think the former values represented), to facilitate a quick look-up.